source/_components/sensor.filter.markdown

```diff
@@ -8,14 +8,14 @@ comments: false
 sharing: true
 footer: true
 ha_category: Sensor
-ha_release: 0.65
+ha_release: 0.66
```

Comment:
I'm not sure, but isn't this variable supposed to say when this was added? I *think* it's also used in the ["Added in ..."](https://home-assistant.io/components/#version/0.65) sections.

Reply:
Corrected.